src/main/scala/com/bwsw/cloudstack/vault/server/cloudstack/CloudStackService.scala

```diff
@@ -108,16 +108,24 @@ class CloudStackService(сloudStackTaskCreator: CloudStackTaskCreator,
   def getAccountIdByVmId(vmId: UUID): UUID = {
     logger.debug(s"getAccountIdByVmId(vmId: $vmId)")
 
-    val accountName = jsonSerializer.deserialize[VirtualMachinesResponse](
-      getEntityJson(vmId.toString, сloudStackTaskCreator.idParameter, Command.ListVirtualMachines)
+    val virtualMachine = jsonSerializer.deserialize[VirtualMachinesResponse](
```

Comment:
why do you use both 'VirtualMachine' and its acronym 'VM'?

Reply:
fixed